_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -129,6 +129,14 @@ This `object` element which embeds an audio resource has an empty [accessible na
 <object aria-labelledby="download" data="/test-assets/moon-audio/moon-speech.mp3"></object>
 ```
 
+#### Failed Example 5
+
+This `object` element has an empty [accessible name][] as the `img` attribute has not been set correctly.
```

Comment:
The description needs to be clearer. What `img` attribute has not been set correctly?

Reply:
I don't believe that this example actually fails anything regarding WCAG, though it's a very strange example. The `object` element has no `data` attribute, so it falls back to the `img`. The `img` has no `src`, which causes browsers to visibly show the `alt` text. The `alt` is exposed to ATs and to sighted users. It's "broken", but not with regards to accessibility...

Reply:
I think I might need help on making this work - I will raise it in our call today!